api_bonus.yaml

```
@@ -0,0 +1,144 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Bonus API
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/bonus/vacanze/request":
```

since this is a request for eligibility, not a request for the actual bonus, I'd make it explicit

```suggestion
  "/bonus/vacanze/eligibility":
```